common/country_formation/dvg_formables_europe.txt

```diff
@@ -157,12 +157,15 @@ PLC = { #Poland-Lithuania
 		STATE_VOLHYNIA
 		STATE_WEST_GALICIA
 		STATE_WEST_PRUSSIA
+		STATE_MINSK
+		STATE_KIEV
 	}
 
 	required_states_fraction = 1
 
 	possible = {
-		has_technology_researched = pan-nationalism #Replace with something like a journel entry requirement
+		has_technology_researched = pan-nationalism 
+		has_variable = can_unite_plc
```

Comment:
![image](https://user-images.githubusercontent.com/116951759/203149324-f51d43e1-1618-44b3-83fc-78c9df038ce9.png)
If you want to use a variable as condition. You need to add a custom_tooltip describing it.